hashtable.csv

```diff
@@ -10,3 +10,14 @@ code, name
 0x19, shake-256
 0x40, blake2b
 0x41, blake2s
+0x55, md5
+0x60, xxHash32
+0x61, xxHash64
+0x62, Spooky32
+0x63, Spooky64
+0x64, Spooky128
+0x65, Murmur3A
+0x66, Murmur3C
+0x67, Murmur3F
+0x75, crc32
+0x76, crc32-C
```

Comment:
It would be fantastic to have "multihash" being an IETF supported document, but adding hashes to the list like this would make it rather impossible.

The good thing by having the IANA registry is that algorithms in the registry can be clearly marked as being deprecated. This has happened to MD5 in almost all present algorithm registries. See for example here:
http://www.iana.org/assignments/dns-sec-alg-numbers/dns-sec-alg-numbers.xhtml

If going for your own registry, please have one or more extra fields describing these properties to the algorithms, with a possible source reference.

So you can add "status", and "reference" to the csv table.